extensions/interactions/GraphInput/directives/graph-utils.service.ts

```diff
@@ -117,8 +117,8 @@ export class GraphUtilsService {
     return false;
   }
 
-  constructAdjacencyMatrix(graph: GraphAnswer): number[][] {
-    var adjMatrix = [];
+  constructAdjacencyMatrix(graph: GraphAnswer): (number | null)[][] {
```

Comment:
Can you separate `(number | null)[][]` into its own type definition, so that it is more reusable?

Reply:
I added a `AdjacencyMatrix` type to this file, let me know if I should put it somewhere that's reusable.